config/env.yml.template

```diff
@@ -3,11 +3,16 @@ defaults: &defaults
   OMNIAUTH_CLIENT_SECRET: 'omniauth_client_secret'
   OMNIAUTH_CLIENT_ID: 'omniauth_client_id'
   DEVISE_SECRET_KEY: 'devise_secret_key'
+  FACEBOOK_APP_ID: '1234'
+  SHARE_PROGRESS_API_KEY: '1234'
 
 # AWS config variables - you get these from https://console.aws.amazon.com/ or https://console.aws.amazon.com/iam/
   AWS_ACCESS_KEY_ID: 'Access Key ID'
   AWS_SECRET_ACCESS_KEY: 'Secret Access Key'
   AWS_REGION: 'Your region in AWS'
+
+  DEFAULT_PLUGIN_REGISTRATION: true
```

Comment:
I would drop this too unless we expect most people to want that behavior in development.